ampd/src/queue/queued_broadcaster.rs

```diff
@@ -281,13 +282,14 @@ mod test {
             .expect_broadcast()
             .once()
             .returning(move |msgs| {
-                assert!(msgs.len() == tx_count);
+                assert_eq!(msgs.len(), tx_count);
 
                 Ok(TxResponse::default())
             });
+        let desired_interval = interval(Duration::from_millis(100));
 
         let queued_broadcaster =
```

Comment:
we should do the same thing here as in the other cases: pause the test, get rid of the initial tick, queue msgs, then advance time instead of sleeping

Reply:
Sure. Also removed the sleep since we have a the `drop(client)`.